gobblin-config-management/gobblin-config-core/src/main/java/gobblin/config/store/hdfs/SimpleHDFSConfigStoreFactory.java

```diff
@@ -0,0 +1,108 @@
+package gobblin.config.store.hdfs;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileStatus;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+
+import gobblin.config.store.api.ConfigStoreCreationException;
+import gobblin.config.store.api.ConfigStoreFactory;
+
+
+/**
+ * An implementation of {@link ConfigStoreFactory} for creating {@link SimpleHDFSConfigStore}s. This class only works
+ * the physical scheme {@link #HDFS_SCHEME_NAME}.
+ */
+public class SimpleHDFSConfigStoreFactory implements ConfigStoreFactory<SimpleHDFSConfigStore> {
```

Comment:
ConfigStoreFactory need to handle the URI without authority / config store root, for example 
etl-hdfs:///datasets/a1/a2